src/components/pages/contact/contact-form.tsx

```diff
@@ -1,148 +1,60 @@
 import React, { useState } from 'react';
-import { useForm } from 'react-hook-form';
-import {
-  ButtonText,
-  ErrorMessageSend,
-  InputField,
-  SendButton,
-  SentButton,
-  Sup,
-} from '../../legacy/form/controls';
-import { Grid, GridItem } from '../../legacy/grid/grid';
-import { Link } from '../../legacy/links/links';
-import { CaptionText } from '../../legacy/typography';
+import { SectionHeader } from '../../content/section-header/section-header';
 import { Trans, useTranslation } from 'gatsby-plugin-react-i18next';
 import styled from 'styled-components';
-import { SIMPLE_EMAIL_PATTERN } from '../../legacy/form/constants';
-import { CheckmarkIcon } from '../../legacy/icons/form-icons/checkmark';
-import { RightArrowIcon } from '../../legacy/icons/form-icons/right-arrow';
-import { HoneypotField } from './honeypot';
+import { Link } from '../../legacy/links/links';
+import { Form } from './form';
+import { theme } from '../../layout/theme';
+import { Button } from '../../ui/buttons/button';
 
-type RequestStatus = 'pending' | 'submitting' | 'success' | 'error';
-const API_ENDPOINT = '/api/contact-form';
+const StyledLink = styled(Link)`
+  color: ${theme.palette.text.link.default};
 
-const StyledCaptionText = styled(CaptionText)`
-  color: inherit;
+  &:hover {
+    border-bottom: 1px solid ${theme.palette.text.link.default};
+  }
 `;
-interface FormData {
-  name: string;
-  email: string;
-  message: string;
-}
-
-export const ContactForm: React.FC = () => {
-  const { t } = useTranslation();
-  const [requestStatus, setRequestStatus] = useState<RequestStatus>('pending');
-  const {
-    register,
-    handleSubmit,
-    formState: { errors },
-  } = useForm();
-
-  const onSubmit = (formData: FormData) => {
-    setRequestStatus('submitting');
```

Comment:
@hofmann-felix this is crucial. Please bring back that combined with the disabled button to prevent multiple submissions while the request is loading